### DISCUSSION TITLE
Can not see query parameter description in redoc?

### DISCUSSION THREAD

**Discussion by heindrickdumdum0217** [fastapi/fastapi#6886] [Questions]
I mapped query parameters with pydantic model.

async def get(params: XXXModel = Depends()):

It works perfectly and I can see query parameters details in redoc.
But only one missing is I can't see query parameter description.
Of course I added description to filed in model definition.
I can see name, type, required or not, max length, min length etc.

**Comment by varun5001** [comment]
@heindrickdumdum0217 can you share your fastapi version and your model

**Comment by heindrickdumdum0217** [comment]
fastapi==0.63.0

And model is pretty simple.
Just country code and language.

class RequestModel(BaseModel):
    cc: str = Field(
        ...,
        title="country code",
        description="The two-letter country code",
        min_length=2,
        max_length=2,
        example=["US"]
    )
    lang: str = Field(
        None,
        title="language",
        description="language",
        min_length=2,
        max_length=2
    )

**Comment by tiangolo** [comment]
Sorry, I can't help you if you don't follow the issue template and provide more information.

If you still have problems please create a new issue and follow the template step by step.

Given that, I'm gonna close this one.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
